nengo_deeplearning/utils.py

```diff
@@ -292,8 +292,11 @@ def minibatch_generator(inputs, targets, minibatch_size, shuffle=True,
     rng : :class:`~numpy:numpy.random.RandomState`, optional
         random number generator
 
-    Yields
-    ------
+    Returns
+    -------
+
+    A generator that yeilds:
```

Comment:
So it was! I upgraded and the warning goes away, so I'll push a commit reverting this change.